Dockerfile.selenium

```diff
@@ -0,0 +1,9 @@
+FROM selenium/standalone-firefox:2.53.0
```

Comment:
may find the `-debug` tag suffix helpful to allow VNC connections to it - bigger image but more introspection